config/default.yml

```diff
@@ -519,8 +519,10 @@ Style/HashSyntax:
     - ruby19
     - ruby19_no_mixed_keys
     - hash_rockets
-# Force hashes that have a symbol value to use hash rockets
+  # Force hashes that have a symbol value to use hash rockets
   UseHashRocketsWithSymbolValues: false
+  # Do not suggest { a?: 1 } over { :a? => 1 } in ruby19 style
+  AllowHashRocketsWithSymbolValuesEndingInNonAlnum: true
```

Comment:
haha, yeah definitely, I struggled with it! What about 
1. `AllowNonAlnumEndingSymbolHashRockets`
2. `AllowHashRocketsForPunctuationEndingSymbols`
3. `EndingPunctuationAllowedForHashRockets`

contrasting with the current
- `AllowHashRocketsWithSymbolValuesEndingInNonAlnum`

Ok, I'll rebase and make it false by default